auslib/test/admin/views/test_rules.py

```diff
@@ -795,7 +796,9 @@ def testAddScheduledChangeExistingRule(self):
         }
         self.assertEquals(db_data, expected)
 
-    def testAddScheduledChangeNewRule(self):
+    @mock.patch("time.time")
+    def testAddScheduledChangeNewRule(self, time):
+        time.return_value = 300
```

Comment:
Was curious if this could be done in a one-liner, and it turns out yes:
  `@mock.patch("time.time", mock.MagicMock(return_value=300))`
and then the function definition needn't be modified.